### PR TITLE
Latest IBLI Updates

### PR DESCRIPTION
**Pull request by mldk-tech**
#71

### CODE REVIEW

**Comment by mldk-tech** [comment]
You see the last ibli updates.

ibli last update

**Comment by mldk-tech** [comment]
You see the last ibli updates.

ibli last update

**Comment by NofarGizra** [comment]
@DavidGizra 
Tabs "Partners" and "Events" look good!
But what happened to the "News" tab? It is all messed up... (see screenshots)

**Comment by NofarGizra** [comment]
@DavidGizra 

Before:

iblinewsupdates2

After:

iblinewsupdates

File: ibli/modules/custom/ibli_general/ibli_general.install [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+/**
+ * Enable module blog.
+ */
+function ibli_general_update_7000() {
```
Comment (NofarGizra) [inline ibli/modules/custom/ibli_general/ibli_general.install]: Remove. There is no module named `last_updates`.

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<div class="block-header">
+  <h2>
+    <span class="title">LATEST IBLI UPDATES</span><span class="decoration"></span><span class="decoration"></span><span class="decoration"></span>
```
Comment (NofarGizra) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php]: t()

**Comment by mldk-tech** [comment]
You can see all 3 tabs after rebuild the project.

partners
events
news

File: ibli/modules/custom/ibli_general/ibli_general.pages_default.inc [hunk 3]
```diff
@@ -52,6 +52,7 @@ function ibli_general_default_page_manager_pages() {
       'left' => NULL,
       'right' => NULL,
       'bottom' => NULL,
+      'last_updates' => NULL,
```
Comment (NofarGizra) [inline ibli/modules/custom/ibli_general/ibli_general.pages_default.inc]: Remove new pane, use the right pane.

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
+<div class="block-header">
+  <h2>
+    <span class="title"><?php echo t("LATEST IBLI UPDATES") ?></span><span class="decoration"></span><span class="decoration"></span><span class="decoration"></span>
+  </h2>
+</div>
+<ul class="nav nav-tabs">
+  <li class="active"><a href="#partners" data-toggle="tab">Partners</a></li>
```
Comment (mldk-tech) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php]: t()

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+/**
+ * Edit form.
+ */
+function ibli_general_last_updates_type_edit_form($form, &$form_state) {
+  return $form;
+}
+
+function ibli_general_last_updates_content_type_edit_form_submit($form, &$form_state) {
```
Comment (mldk-tech) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc]: Remove function

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  'description' => t('Display Last Updates'),
+  'category' => t('ibli'),
+  'hook theme' => 'ibli_general_last_updates_content_type_theme',
+);
+
+/**
+ * Render callback.
+ */
+function ibli_general_last_updates_content_type_render($subtype, $conf, $args, $context) {
+
```
Comment (mldk-tech) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc]: remove space

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  $variable = array();
```
Comment (mldk-tech) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc]: remove

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+  $block = new stdClass();
+  $block->module = 'ibli_general';
+  $block->content = theme('ibli_general_last_updates', $variable);
```
Comment (mldk-tech) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc]: remove second variable

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+function ibli_general_last_updates_content_type_edit_form_submit($form, &$form_state) {
+  return TRUE;
+}
+
+/**
+ * Delegated hook_theme().
+ */
+function ibli_general_last_updates_content_type_theme(&$theme, $plugin) {
+  $theme['ibli_general_last_updates'] = array(
+    'variable' => array(
```
Comment (mldk-tech) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc]: remove

File: ibli/themes/custom/bootstrap_subtheme/template.php [hunk 10]
```diff
@@ -42,3 +42,12 @@ function bootstrap_subtheme_preprocess_ibli_homepage(&$variables) {
   );
   drupal_add_js($setting, array('type' => 'setting'));
 }
+
+/**
+ * Preprocess last updates.
+ */
+function bootstrap_subtheme_preprocess_ibli_general_last_updates(&$variables) {
+
```
Comment (mldk-tech) [inline ibli/themes/custom/bootstrap_subtheme/template.php]: remove space

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php [hunk 11]
```diff
@@ -0,0 +1,78 @@
+<div class="block-header">
+  <h2>
+    <span class="title"><?php echo t("LATEST IBLI UPDATES") ?></span><span class="decoration"></span><span class="decoration"></span><span class="decoration"></span>
```
Comment (NofarGizra) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php]: All strings must be within `'` and not `"`

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php [hunk 12]
```diff
@@ -0,0 +1,78 @@
+<div class="block-header">
+  <h2>
+    <span class="title"><?php echo t('LATEST IBLI UPDATES') ?></span><span class="decoration"></span><span class="decoration"></span><span class="decoration"></span>
```
Comment (NofarGizra) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php]: Always use `print`, never use `echo`.

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+<ul class="nav nav-tabs">
+  <li class="active"><a href="#partners" data-toggle="tab"><?php echo('Partners'); ?></a></li>
+  <li><a href="#events" data-toggle="tab"><?php echo t('Events'); ?></a></li>
+  <li><a href="#news" data-toggle="tab"><?php echo t('News'); ?></a></li>
+</ul>
+<div class="tab-content">
+  <div class="tab-pane active" id="partners">
+    <div class="media">
+      <a class="pull-left" href="#">
+        <img class="media-object" src="<?php print $images_path; ?>/eu.png" alt="Blog Message">
```
Comment (NofarGizra) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last-updates.tpl.php]: All `img` tags must end with a slash `/`.
For example:

```
<img src="photo.png" />
```

File: ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+function ibli_general_last_updates_type_edit_form($form, &$form_state) {
+  return $form;
+}
+
+/**
+ * Delegated hook_theme().
+ */
+function ibli_general_last_updates_content_type_theme(&$theme, $plugin) {
+  $theme['ibli_general_last_updates'] = array(
+
```
Comment (NofarGizra) [inline ibli/modules/custom/ibli_general/plugins/content_types/last_updates/last_updates.inc]: Remove space.

**Comment by NofarGizra** [comment]
Merged.

**Pull request closed without merging**